generate_report_excels.py

```
@@ -0,0 +1,836 @@
+            logging.debug(f"Audit logs response: {response.json()}")
+            return response.json().get("content", [])
+        else:
+            logging.error(f"Error fetching audit logs: {response.status_code}")
+            raise RequestException(
+                f"Error fetching audit logs. Status code: {response.status_code}"
+            )
+    except RequestException as e:
+        logging.error(f"Request exception while fetching audit logs: {e}")
+        raise RequestException(
+            f"Error fetching audit logs. {e}"
+        )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)